test/test_podman_baseline.sh

```diff
@@ -78,7 +78,71 @@ echo $image
 ########
 # Run container and display contents in /etc
 ########
-podman run $image ls -alF /etc
+podman run --rm $image ls -alF /etc
+
+########
+# Test networking, bind mounting a file, stdin/stdout redirect
+########
+txt1="Hello, Podman"
+echo "$txt" > /tmp/hello.txt
+podman run -d --name myweb -p 8080:80 -w /var/www -v /tmp/hello.txt:/var/www/index.txt busybox httpd -f -p 80
+echo $txt | podman exec -i myweb sh -c "cat > /var/www/index2.txt"
+txt2=$( podman exec myweb cat /var/www/index2.txt )
+[ "x$txt1"=="x$txt2" ] && echo "PASS"
+txt2=$( podman run --rm --net host busybox wget -qO - http://localhost:8080/index.txt )
+[ "x$txt1"=="x$txt2" ] && echo "PASS"
+txt2=$( podman run --rm --net host busybox wget -qO - http://localhost:8080/index2.txt )
+[ "x$txt1"=="x$txt2" ] && echo "PASS"
+# poadman run --rm --net container:myweb --add-host myweb:127.0.0.1 busybox wget -qO - http://myweb/index.txt
+rm /tmp/hello.txt
+podman stop myweb
+podman rm myweb
+
+########
+# pull and run many containers in parallel, test locks ..etc.
+########
+podman rmi docker.io/library/busybox:latest > /dev/null || :
+for i in `seq 10`
+do ( podman run -d --name b$i docker.io/library/busybox:latest busybox httpd -f -p 80 )&
+done
+echo -e "\nwaiting for creation...\n"
+wait
+echo -e "\ndone\n"
+# assert we have 10 running containers
+count=$( podman ps -q  | wc -l )
+[ "x$count" == "x10" ] || { echo "was expecting 10 running containers"; exit -1; }
+
+for i in `seq 10`; do ( podman stop -t=1 b$i; podman rm b$i )& done
+echo -e "\nwaiting for deletion...\n"
+wait
+echo -e "\ndone\n"
+# assert we have 0 running containers
+count=$( podman ps -q  | wc -l )
+[ "x$count" == "x0" ] || { echo "was expecting 0 running containers"; exit -1; }
+
+
+########
+# run many containers in parallel for an existing image, test locks ..etc.
+########
+podman pull docker.io/library/busybox:latest > /dev/null || :
+for i in `seq 10`
+do ( podman run -d --name c$i docker.io/library/busybox:latest busybox httpd -f -p 80 )&
+done
+echo -e "\nwaiting for creation...\n"
+wait
+echo -e "\ndone\n"
+# assert we have 10 running containers
+count=$( podman ps -q  | wc -l )
+[ "x$count" == "x10" ] || { echo "was expecting 10 running containers"; exit -1; }
+
+for i in `seq 10`; do ( podman stop -t=1 c$i; podman rm c$i )& done
+echo -e "\nwaiting for deletion...\n"
+wait
+echo -e "\ndone\n"
+# assert we have 0 running containers
+count=$( podman ps -q  | wc -l )
+[ "x$count" == "x0" ] || { echo "was expecting 0 running containers"; exit -1; }
```

Comment:
I forgot to mention/ask for this yesterday, my apologies.  I like this test to not stop if there's an error unless the '-e' param is passed in.  For this line and the others where you exit on error, can you rework it slightly  to something like:

```
[ "x$count" == "x0" ] || { echo "was expecting 0 running containers";-}
if [ "$showerror" eq 1 ]; then
   [ "x$count" == "x0" ] || { exit -1; }
fi
```

Reply:
I'll fix that